### ISSUE TITLE
[FEAT]: New UI look for mobile

### ISSUE THREAD

**Issue by tofuisme** [nova-video-player/aos-AVP#818]
### Description

It has been a while nova video player doesn't have new ui update . Please add material you theme options and more fined tuned looks for mobile.

As for example like this 
![Uploading Screenshot_20230206-133742_HDO Box.png…]()

![Screenshot_20230206-134045_Aniyomi](https://user-images.githubusercontent.com/94361006/216906769-c999b465-fe22-4c2e-b14d-217a95bb2e08.png)

**Comment by realFPS** [comment]
I agree too, I love this app.I also have more suggestions
1. Grid customization 
2. A toggle to disable external options from the sidebar if not needed
3. If possible a next episode button in the circled area. The current process is too lengthy if using external player (First go to played episode page, then to all episodes then select next episode then select play, 4 taps)
![image](https://user-images.githubusercontent.com/77321589/218325563-fc367bb5-888e-442a-999e-15bd1f387c96.png)
4. Anilist Sync?😥

**Comment by liyunkkk** [comment]
Yes. I also support it. Hope to add a new UI, similar to MD3 style design, it will look more beautiful！

**Comment by ngocanhtve** [comment]
> If possible a next episode button in the circled area. The current process is too lengthy if using external player (First go to played episode page, then to all episodes then select next episode then select play, 4 taps)

You don't need that button, just swipe right to left to go to the next episode.
